lib/Runtime/Language/InterpreterLoop.inl

```diff
@@ -25,6 +25,126 @@
 #define INTERPRETERPROFILE 0
 #define PROFILEDOP(prof, unprof) unprof
 #endif
+
+//two layers of macros are necessary to get arguments to the invocation of the top level macro expanded.
```

Comment:
Didn't quite understand the reason for two layers. Isn't one macro showing up in the call stack ?